http_requests_base.py

```diff
@@ -150,6 +156,13 @@ def _process_response(self, response, signal):
                     sig._resp = response.__dict__
                 except:
                     self.logger.warning("Response failed to save to signal")
+
+            self.logger.debug("{} request to {} returned with response code: "
+                              "{}. Response: {}"
+                              .format(self.http_method(),
+                                      self.url(signal),
+                                      response.status_code,
+                                      [res.to_dict() for res in result]))
```

Comment:
I don't think we want all the signals in the logs. Just the response from the request. Probably want to log something like `response.json()` if it's json or `response.text`. I'm just looking at the code above here to see what it's doing. Maybe even `response.__dict__` to get everything. It's possible that doesn't turn into a string well and would print weird in the logs. Check it out first if you consider that route.